static/docs/commands-reference/checkout.md

```diff
@@ -103,36 +103,20 @@ be pulled from a remote cache using `dvc pull`.
 
 ## Examples
 
-To explore `dvc checkout` let's consider a simple workspace with several
-[stages](/doc/commands-reference/run), and a few Git tags. Then with
-`git checkout` and `dvc checkout` we can see what happens as we shift from tag
-to tag.
+Let's employ a simple workspace with some stages and a few Git tags, such as our
```

Comment:
`some code, data, ML models, and pipelines` is better than `some stages`

Reply:
Sure. Updated in f1dc617